colly.go

```diff
@@ -688,6 +732,50 @@ func (c *Collector) handleOnHTML(resp *Response) {
 	}
 }
 
+func (c *Collector) handleOnXML(resp *Response) {
+	if (!strings.Contains(strings.ToLower(resp.Headers.Get("Content-Type")), "html") && !strings.Contains(strings.ToLower(resp.Headers.Get("Content-Type")), "xml")) || len(c.xmlCallbacks) == 0 {
+		return
+	}
+
+	if !strings.Contains(strings.ToLower(resp.Headers.Get("Content-Type")), "html") {
```

Comment:
This condition is a bit unclear for me. Why do you negate the expression? As I understand this block is executed if the content-type doesn't contain "html".

Reply:
Oh my! Good catch.